app/models/weather_request.rb

```diff
@@ -1,9 +1,11 @@
 class WeatherRequest < ApplicationRecord
   belongs_to :location
 
-  STATUSES = [PROCESSING = 'Processing',
-              ERROR      = 'Error',
-              SUCCESS    = 'Success'].freeze
+  STATUSES = [
+    PROCESSING = 'Processing',
+    ERROR      = 'Error',
+    SUCCESS    = 'Success'
+  ].freeze
```

Comment:
Still not sure of an easier way to define this, so just making it nicer for now.